MapboxCoreNavigation/RouteController.swift

```diff
@@ -115,6 +115,11 @@ open class RouteController: NSObject {
     }
     
     /**
+     If true, the `RouteController` was initialized with a `SimulatedLocationManager` as the `NavigationLocationManager`. This `Bool` is used to notifiy the user they are currently viewing the `NavigationViewController` in simulation mode.
+     */
+    public var isSimulationModeEnabled = false
```

Comment:
There's no need to add this to core. Let's just do a `if routeController.locationManager is SimulationLocationManager` in the UI part.